src/main/java/com/softserve/service/CategoryService.java

```diff
@@ -31,5 +32,10 @@ public List<Category> findByIdParentCategoryIsNull(){
 	public List<Category> findByCategoryParent(Integer idCategoryParent){
 		return this.categoryRepository.findByIdParentCategory(idCategoryParent);
 	}
+
+	public Optional<Category> findById(Integer id) {
```

Comment:
watch out the indentation on this one, formatting is important

Reply:
Hi Cesar, sorry for late. In my IDE the identation is fine, I can't add the image resource but is fine right now!!